patches/master_patch.patch

```diff
@@ -159,6 +159,159 @@ index 751043efd8038573b2870dc60e5d7ad737887d5f..321971b975659028005aabd8a9163ac8
            "safe_browsing::mojom::SafeBrowsing",
            "spellcheck::mojom::SpellCheckHost",
            "spellcheck::mojom::SpellCheckPanelHost",
+diff --git a/chrome/browser/download/download_target_determiner.cc b/chrome/browser/download/download_target_determiner.cc
+index 4f4dadd98fd997e9659d10e6989916dbb0bb899e..6c90abf8967dd01226c72c1f2ce77feb2a025680 100644
+--- a/chrome/browser/download/download_target_determiner.cc
++++ b/chrome/browser/download/download_target_determiner.cc
+@@ -16,7 +16,6 @@
+ #include "base/threading/thread_task_runner_handle.h"
+ #include "base/time/time.h"
+ #include "chrome/browser/download/chrome_download_manager_delegate.h"
+-#include "chrome/browser/download/download_crx_util.h"
+ #include "chrome/browser/download/download_prefs.h"
+ #include "chrome/browser/download/download_stats.h"
+ #include "chrome/browser/history/history_service_factory.h"
+@@ -92,7 +91,7 @@ DownloadTargetDeterminer::DownloadTargetDeterminer(
+     DownloadPrefs* download_prefs,
+     DownloadTargetDeterminerDelegate* delegate,
+     const CompletionCallback& callback)
+-    : next_state_(STATE_GENERATE_TARGET_PATH),
++    : next_state_(STATE_CHECK_DOWNLOAD_URL),
+       confirmation_reason_(DownloadConfirmationReason::NONE),
+       should_notify_extensions_(false),
+       create_target_directory_(false),
+@@ -278,15 +277,7 @@ DownloadTargetDeterminer::Result
+   DCHECK(!virtual_path_.empty());
+ 
+   next_state_ = STATE_RESERVE_VIRTUAL_PATH;
+-
+-  if (!should_notify_extensions_ ||
```

Comment:
can you provide some explanation for these? This is adding a lot of patches and it doesn't seem like they should all be necessary